charts/corda/templates/workers.yaml

```
@@ -1,25 +1,82 @@
+{{- define "cluster-ip-service" }}
```

Function names should be "namespaced" to avoid collisions when combining charts. Also, this function should be defined in `corda-lib` so that it can be reused in the Enterprise Helm chart.
```suggestion
{{- define "corda.cluster-ip-service" }}
```